src/cli.ts

```diff
@@ -9,18 +9,15 @@ const {
   onlyTemplate,
   onlyTypeScript,
   excludeDir,
+  _,
 } = minimist(process.argv.slice(2));
-
-if (!workspace) {
-  throw new Error("--workspace is required");
-}
-
 const cwd = process.cwd();
 
 check({
-  workspace: path.resolve(cwd, workspace),
+  workspace: path.resolve(cwd, workspace || "."),
```

Comment:
Wouldn't it be possible to reuse `cwd` for this?

```suggestion
  workspace: path.resolve(cwd, workspace || cwd),
```

It would be more readable to keep the `!workspace` block, but move it after the `cwd` assignment:

```js
const cwd = process.cwd();

if (!workspace) {
  workspace = cwd;
}
```